kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/ZookeeperConf.scala

```diff
@@ -58,12 +59,14 @@ object ZookeeperConf {
 
   val ZK_DATA_DIR: ConfigEntry[String] = buildConf("kyuubi.zookeeper.embedded.data.dir")
     .doc("dataDir for the embedded zookeeper server where stores the in-memory database" +
-      " snapshots and, unless specified otherwise, the transaction log of updates to the database.")
+      " snapshots and, unless specified otherwise, the transaction log of updates to the database." +
+      " If it is a relative path, it is resolved relative to KYUUBI_HOME.")
```

Comment:
```suggestion
      " snapshots and, unless specified otherwise, the transaction log of updates to the" +
      " database. If it is a relative path, it is resolved relative to KYUUBI_HOME.")
```